components/dashboard/NotificationCard.js

```diff
@@ -0,0 +1,62 @@
+import { makeStyles } from '@material-ui/styles';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { useAuth } from '../Auth';
+
+const useStyles = makeStyles(theme => ({
+  card: {
+    position: 'fixed',
+    zIndex: '100',
+    top: 'auto',
+    bottom: theme.spacing(2),
+    left: theme.spacing(2),
+    maxWidth: theme.spacing(50),
+    borderColor: theme.palette.primary.main,
+    borderWidth: 'medium',
+  },
+}));
+
+export default function NotificationCard() {
+  const classes = useStyles();
+  const { user, userDocRef } = useAuth();
+  const showSurvey = user.isCovidSurveyShown == null || !user.isCovidSurveyShown;
```

Comment:
Can juts be `!user.isCovidSurveyShown` (will catch anything "falsy" -- undefined, null, 0, etc.)

Reply:
done. thanks for calling it out.